client/features/navbar/Navbar.js

```diff
@@ -1,15 +1,16 @@
-import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { Link, useNavigate } from 'react-router-dom';
-import { logout } from '../../app/store';
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Link, useNavigate } from "react-router-dom";
+import { logout } from "../../app/store";
+import { Button, Badge } from "react-bootstrap";
 
 const Navbar = () => {
-  const isLoggedIn = useSelector((state) => !!state.auth.me);
+  const isLoggedIn = useSelector((state) => !!state.auth.me.id);
```

Comment:
does this cause an error when the user isn't logged in?

Reply:
I'll delete the .id part! I had to restore this file to fix git issues so it reverted it back to the previous code, and I forgot to remove it